guides/common/modules/proc_calling-the-api-in-curl.adoc

```diff
@@ -0,0 +1,196 @@
+[id="calling-the-api-in-curl"]
+= Calling the API in curl
+
+You can use `curl` with the {Project} API to perform various tasks.
+
+{ProjectName} requires the use of HTTPS, and by default a certificate for host identification.
+If you have not added the {ProjectServer} certificate as described in xref:ssl-authentication-overview[], then you can use the `--insecure` option to bypass certificate checks.
+
+For user authentication, you can use the `--user` option to provide {Project} user credentials in the form `--user _username:password_` or, if you do not include the password, the command prompts you to enter it.
+To reduce security risks, do not include the password as part of the command, because it then becomes part of your shell history.
+Examples in this section include the password only for the sake of simplicity.
+
+Be aware that if you use the `--silent` option, `curl` does not display a progress meter or any error messages.
```

Comment:
Like this?
```suggestion
Be aware that if you use the `--silent` option, `curl` does not display a progress meter or any error messages.
You can use `--show-error` together with `--silent` to display error messages only.
```
I don't know. This combo just disables the progress meter if I'm reading the manpage correctly. Then the user can just use `--no-progress-meter`. How about we just refer them to curl help instead of adding options?
```suggestion
Be aware that if you use the `--silent` option, `curl` does not display a progress meter or any error messages.
For more information about command-line output options, see `curl --help verbose`.
```

Reply:
Your suggestions make me wonder if there's a benefit of using "curl --silent --show-error" over "curl". I'll ask around and let you know.

not blocking your PR either way.